api/versions.py

```
@@ -0,0 +1,95 @@
+from rest_framework import viewsets, status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from records_versioning.models import RecordVersion
+from records_versioning.services import revert_to_version
+
+class VersionViewSet(viewsets.ViewSet):
+    def list(self, request, resource_type=None, resource_id=None):
```

Ruff (0.12.2)

Missing return type annotation for public function `list`

(ANN201)